### PR TITLE
Upgrade WDKClient and resolve breaking change from WDKClient refactor

### PR DESCRIPTION
**Pull request by jernestmyers**
This PR upgrades the WDKClient package and resolves a file path that was affected by the deletion of `MoveAfterRefactor`.

### CODE REVIEW

File: Site/package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "apidb",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (jtlong3rd) [inline Site/package.json:3]: Just to clarify: the `*Website` and `install` repos aren't published to npm, so their versions don't have to be incremented

File: Site/package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
     "@types/cytoscape": "3.2.9",
     "@veupathdb/multi-blast": "^1.0.5",
     "@veupathdb/preferred-organisms": "^1.1.9",
-    "@veupathdb/wdk-client": "^0.5.6",
+    "@veupathdb/wdk-client": "^0.5.12",
```
Comment (jtlong3rd) [inline Site/package.json:10]: You'll also want to commit the new `yarn.lock` file. Speaking of which, what command did you use to upgrade `@veupathdb/wdk-client`?

**Review by jtlong3rd — COMMENTED** [review]
The fix for the breaking change looks good! See my inline comments.

File: Site/package.json (hunk 2 above)
Comment (jernestmyers) [reply Site/package.json:10]: I used `yarn upgrade @veupathdb/wdk-client@latest`

Comment (jtlong3rd) [reply Site/package.json:10]: Great, then that command should have produced a new `yarn.lock`.

**Review by jtlong3rd — APPROVED** [review]
🎉

**Pull request merged**